app/lib/server/functions/notifications/email.js

```diff
@@ -162,11 +162,14 @@ export function getEmailData({
 
 	if (sender.emails?.length > 0) {
 		const [senderEmail] = sender.emails;
-		email.headers['Reply-To'] = generateNameEmail(username, senderEmail.address);
+
+		if (settings.get('Direct_Reply_ReplyTo_Enable')) {
+			email.headers['Reply-To'] = generateNameEmail(username, senderEmail.address);
+		}
 	}
 
 	// If direct reply enabled, email content with headers
-	if (settings.get('Direct_Reply_Enable')) {
+	if (settings.get('Direct_Reply_Enable') && settings.get('Direct_Reply_ReplyTo_Enable')) {
```

Comment:
In case Direct Reply is enabled, the `Reply-To` header should be always set, since that is the only way we can make sure to receive a reply from the notification email, this new setting should be only applied to regular email notifications (when Direct Reply is turned off :wink: )